cli/flags.go

```diff
@@ -92,6 +92,9 @@ Database server port to connect to.`),
 	forClient(cliflags.HTTPPortName): wrapText(`
 Database server port to connect to for HTTP requests.`),
 
+	forClient(cliflags.HTTPIPAddrName): wrapText(`
+Database server IP address to connect to for HTTP requests.`),
```

Comment:
Ditto for the help message: `Database server port to connect to for HTTP requests.`